### DISCUSSION TITLE
Confused by documentation

### DISCUSSION THREAD

**Discussion by IcyAlmond** [NvChad/base46#360] [Q&A] [answered]
`:h nvui.base46.highlights` shows this section:
<img width="470" alt="{5561800E-096E-4DE7-B1DF-906DC43C92B5}" src="https://github.com/user-attachments/assets/91daf99f-bf7c-4833-bb52-2dca25689b9a">

What makes this part lighten black by 5x? what's the relationship between the 2 and 5.

**Comment by siduck** [comment, marked as answer]
@Latrolage its a typo, fixed
